src/SFA.DAS.Authorization.Mvc/Filters/AuthorizationFilter.cs

```diff
@@ -23,9 +23,16 @@ public async Task OnAuthorizationAsync(AuthorizationFilterContext context)
         {
             var controllerActionDescriptor = (ControllerActionDescriptor)context.ActionDescriptor;
             var dasAuthorizeAttributes = controllerActionDescriptor.GetDasAuthorizeAttributes();
-
+            var staffAuthorizeAttributes = controllerActionDescriptor.GetStaffAuthorizeAttributes();
+           
+                       
             if (dasAuthorizeAttributes.Count > 0)
             {
+                if (staffAuthorizeAttributes.Count > 0)
+                {
+                    return;
```

Comment:
Does this mean that is there is a StaffAuthorise attribute that no authorization checks are performed?

Reply:
Yes for now for this Spike.

Reply:
Isn't that just the same as the AllowAnonymous attribute?